### DISCUSSION TITLE
LXC and SSH login

### DISCUSSION THREAD

**Discussion by Depechie** [tteck/Proxmox#3336] [Q&A] [answered]
What is the actual procedure for the LXC container scripts to be able to connect through SSH?
Generally when installing a LXC through the scripts, you can always open a console to the container through the proxmox UI.

But I like to use my Mac terminal app instead of going through the proxmox web UI.

But what is the user and password default?
Or is that not set and do we need to config something after a LXC script install?

**Comment by Mellowlynx** [comment]
Hi there.

You can enable direct SSH per LXC (https://github.com/tteck/Proxmox/discussions/385), but I SSH intro the host and enter the LXC from there.
`pct enter LXC_ID`

**Reply by tteck** [reply]
Beat me to it. 😜

**Comment by tteck** [comment, marked as answer]
Typically, if SSH is required, one would use the advanced settings to create the LXC, which includes options for setting a root password and enabling SSH.

If you have already created an LXC with the default settings, you can follow the instructions in this discussion: https://github.com/tteck/Proxmox/discussions/385.

**Comment by Depechie** [comment]
Thanks @tteck and @Mellowlynx for the details!!
